pyproject.toml

```
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["flit_core>=3.7"]
+requires = ["flit_core @ git+https://github.com/pypa/flit.git#subdirectory=flit_core"]
```

When Flit 3.11 is released
```suggestion
requires = ["flit_core>=3.11"]
```